mdsuite/calculators/angular_distribution_function.py

```diff
@@ -137,22 +137,23 @@ def __init__(self, **kwargs):
         self.adf_minibatch = None  # memory management for triples generation per batch.
 
         self.analysis_name = "Angular_Distribution_Function"
-        self.x_label = r"$$\text{Angle} / \theta $$"
-        self.y_label = r"$$\text{ADF} / a.u.$$"
+        self.x_label = r"$\text{Angle} / \theta$"
+        self.y_label = r"$\text{ADF} / a.u.$"
 
     @call
     def __call__(
         self,
         batch_size: int = 1,
-        minibatch: int = 50,
+        minibatch: int = -1,
```

Comment:
yep, it is just an if statement check so it can be changed to None, @PythonFZ do you have some preference here, the notation was introduced in the RDF calculator and I have just carried it over.

Reply:
IIRC the reason I used the `-1` over None was, that None does not use minibatching and `-1` does automatically determine the minibatch size. If this is no longer the case I would also set it to None.

Reply:
Sure but I guess the minibatching should actually be determined automatically anyway?

Reply:
Yes, so it should default to `mds.AUTOTUNE` but the option should remain.

Reply:
As we have no autotune capabilities and the minibatching here should theoretically be brought under the relevant memory management already in existence should this be moved to  #476 or another PR altogether? 

The reason initially here for setting it to -1 is that it was defaulted to 50 which failed for any system with less than 50 particles.

Reply:
Sounds good!

Reply:
Opened #480